include/jrl/DatasetBuilder.h

```diff
@@ -43,6 +46,12 @@ class DatasetBuilder {
   /// @brief Adds initial estimate information for a single robot. Can be used incrementally or in bulk.
   void addInitialization(char robot, TypedValues initialization);
 
+  /// @brief Add information for the potential outliers of the dataset
+  void addPotentialOutlierFactors(char robot, std::set<FactorId> potential_outlier_factors);
+
+  /// @brief Add information on the true inliers of the dataset
+  void addOutlierFactors(char robot, std::set<FactorId> outlier_factors);
+
```

Comment:
Might be good to include in the docs that this will override any previously set outliers (if that's the expected behavior)